src/mqttcm_connect.c

```diff
@@ -1452,6 +1504,10 @@ void mqtt_subscribe()
 			else
 			{
 				MqttCMInfo("subscribe to topic %s success\n", topic);
+				MqttCMInfo("Add topic to the list\n");
+				MqttCMInfo("Going inside createComponentSubscribeTopicList function\n");
+				createComponentSubscribeTopicList("Webconfig" ,topic);
+				MqttCMInfo("createComponentSubscribeTopicList function done in mqtt_subscribe\n");
```

Comment:
subscribeFlag set to 1 when one component is subscribed need to be changed. if this is set to 1 after first component, other components will not be able to subscribe based on check at 1487. make this subscribeFlag component specific.